newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationPage.js

```diff
@@ -80,22 +82,26 @@ const getPackColumns = (windowSize: WindowSizeType, isLandscape: boolean) => {
   }
 };
 
-const sortedContentType = [
+const sortedContentType: PrivateAssetPackAssetType[] = [
   'sprite',
   '9patch',
   'tiled',
+  'model3d',
+  'simpleTileMap',
   'particleEmitter',
   'font',
   'audio',
-  'Scene3D::Model3DObject',
```

Comment:
I'm afraid of this, have we not talked about it and decided that the "Full GDevelop type" is safer than inventing our own types like we wrongly did for the rest? cc @D8H

Reply:
I agree, it's a bit of a pain to have to do the translation in the editor.